tests-system/rbt-universal-quantification-semantics/foo.trlc

```diff
@@ -0,0 +1,9 @@
+package Foo
+
+T Bar {
+    a = [-1]
```

Comment:
please add a second number for which the predicate evaluates to `true`, like this:
` a = [-1, 1]`